AccountService.js

```diff
@@ -42,26 +50,41 @@ export class AccountService {
     }
   }
 
+  /**
+   * Takes a url and an email then creates an account for a user.
+   * @return {Object}
```

Comment:
return should be at the bottom of the list with an extra whitespace before it here and throughout.

Reply:
And dying in your beds, many years from now, would you be willing to trade all the days from this day 'til that for one chance, just one chance, to come back here and add a rule to ensure there is whitespace above `@return`? I know I would.